src/test/40-sql-queries.ts

```diff
@@ -0,0 +1,212 @@
+import { expect } from "chai";
+import { Fabric } from "../jsC8";
+import { ArrayCursor } from "../cursor";
+import { C8Error } from "../error";
+import { getDCListString } from "../util/helper";
+import { C8Client } from "../jsC8";
+const C8_VERSION = Number(process.env.C8_VERSION || 30400);
+
+describe("Test Sql query functions", function() {
+  let name = `testdb${Date.now()}`;
+  let fabric: Fabric;
+  const testUrl = process.env.TEST_C8_URL || "https://test.macrometa.io";
+  const collectionName = "products";
+  let dcList: string;
+
+  before(async () => {
+    fabric = new Fabric({ url: testUrl, c8Version: Number(process.env.C8_VERSION || 30400) });
+
+    await fabric.login("demo@macrometa.io", "demo");
```

Comment:
@prashantz-mm lets use `guest@macormeta.io` itself. As we already have `demo` tenant. Down the line we can incorporate reading this from some env